doc/developers-guide/mod_muc_light_developers_guide.md

```diff
@@ -6,24 +6,24 @@ This is an in-depth guide on `mod_muc_light` design decisions and implementation
 Source, header and test suite files
 -------------------------------
 
-All source files can be found in `apps/ejabberd/src`.
+All source files can be found in `src/`.
 
 * `mod_muc_light.erl`
-  Main module. 
-  It implements `gen_mod` behaviour. 
-  It subscribes to some essential hooks and exports several functions, mostly callbacks. 
+  Main module.
+  It implements `gen_mod` behaviour.
+  It subscribes to some essential hooks and exports several functions, mostly callbacks.
   It handles integration with `mod_disco`, `mod_privacy` and `mod_roster`.
-  All operations that take place outside the room (including the room creation) are implemented here. 
+  All operations that take place outside the room (including the room creation) are implemented here.
   Last but not least - this module prevents `service-unavailable` errors being sent when an offline user receives a groupchat message.
 
 * `mod_muc_light_codec.erl`
   A behaviour implemented by modules that translate the MUC Light internal data format to stanzas for clients and vice versa.
   Besides specifying callbacks, it implements generic error encoder function.
 
 * `mod_muc_light_codec_legacy.erl`
-  An implementation of XEP-0045 compatibility mode. 
-  Note that while, some parts of the legacy mode are implemented directly in `mod_muc_light.erl`, the stanza translation takes place here. 
-  It does not utilise the full potential of the MUC Light extension but allows using the standard MUC implementation in XMPP client libraries for prototyping or transition phase. 
+  An implementation of XEP-0045 compatibility mode.
+  Note that while, some parts of the legacy mode are implemented directly in `mod_muc_light.erl`, the stanza translation takes place here.
+  It does not utilise the full potential of the MUC Light extension but allows using the standard MUC implementation in XMPP client libraries for prototyping or transition phase.
```

Comment:
It does not utilise the full potential of the MUC Light extension but allows using the standard MUC implementation in XMPP client libraries for prototyping or the transition phase.